package/utils/util-linux/Makefile

```diff
@@ -35,6 +35,31 @@ define Package/util-linux/Default
   URL:=http://www.kernel.org/pub/linux/utils/util-linux/
 endef
 
+define Package/util-linux-busybox
+$(call Package/util-linux/Default)
+  TITLE:=Full util-linux commands instead of busybox replacements
```

Comment:
Combined with the busybox option above (marked experimental, default n), it allows that instead of compiling the image with busybox applets for which util-linux full versions are available, busybox will be compiled without the applets and only util-linux versions will be present.